plugin-template

```diff
@@ -331,7 +332,11 @@ def main():
         print("\nAn updated plugin template config written to {path}.\n".format(path=file_path))
 
     if "github" in sections:
-        migrate_pytest_pin(plugin_root_dir)
```

Comment:
Can you also delete the function we no longer call?
If we want to dig in the history of the project we still have git.